peachjam/js/components/FindDocuments/SearchResult.vue

```diff
@@ -74,6 +88,13 @@ export default {
       default: false,
     }
   },
+  data (){
+    const documentLabels = JSON.parse(document.querySelector('#data-labels').textContent).documentLabels;
```

Comment:
this will now be done once for every rendered search result, which isn't great. instead, do it at the top level of the main FindDocuments/index.vue component, and then push the documentLabels as state into these components.